1.preprocessing_data/README.md

```diff
@@ -2,52 +2,36 @@
+To confirm that IC was working on the images, we adjusted the brightness/contrast in CellProfiler to see if the contrast between objects and background increased and the illumination looked more even. 
 
-Being able to visualize illumination errors can be hard for some datasets.
-To test if illumination correction is necessary for this dataset, we brightened images using [Fiji](https://imagej.net/software/fiji/).
-Through this, we did observe that there was technical variation in illumination across the images (e.g vignetting), which is a common issue for Cell Painting data (Figure 1).
+## Calculate and save corrected images for all plates
 
-![Raw_to_Bright_fig](example_figs/Raw_to_Bright_fig.png)
+To perform illumination correction and save the corrected images, you will need to:
 
-> Figure 1. Image Comparison: Raw to Brightened. This figure displays the images for each channel in the same well/frame. The channel metadata (red) and Cell Painting assay stain for each channel (blue) is shown to the left of the images. The main indicator of illumination issues is seeing brighter areas, especially in the background. For example, in the `Mito` channel, you can see that on the left side, the area is brighter as compared to the top right.
+### Step 1: Open the CellProfiler GUI
 
-After using PyBaSiC (or BaSiCPy) to correct for illumination, it is hard to tell if there was any change made to the images just by looking at them unedited (Figure 2).
-
-![NotBrightened_Comparison.png](example_figs/NotBrightened_Comparison.png)
-
-> Figure 2. Comparison Between Unedited Raw and Corrected Images. This figure shows that it is very hard for the human eye to see the difference between the raw and corrected images without any editing.
-
-By brightening the images through, there is a noticable difference in illumination between the raw and corrected images where you can tell the illumination is more even in the corrected images (Figure 3)
-
-![Raw_to_Corrected_fig](example_figs/Raw_to_Corrected_fig.png)
+```bash
+# activate the conda env for the repo
+conda activate cfret_data
+# call CellProfiler to start the GUI
+cellprofiler
+```
 
-> Figure 3. Image Comparison: Brightened Raw Images to Brightened Corrected Images. This figure shows the difference in illumination when the raw and corrected images are brightened.
+### Step 2: Open the pipeline in the GUI
 
-## Step 1: Install PyBaSiC
+Drag the `illum.cppipe` file into the GUI to open the pipeline.
+Once the pipeline is open, go to the `Images` module and clear the file list.
+Since Cellprofiler needs absolute paths, you will need to drag the "Images" folder from your local machine in this module to make sure Cellprofiler can find the plates with the images.
```

Comment:
Does the drag and drop features work on all operating systems (mac, linux, and windows?) if not consider providing an alterantive approach on how to load the images.